IntegrationTests/UI.IntegrationTests/GitUICommands/RunCommandTests.cs

```diff
@@ -361,6 +368,25 @@ public void RunCommandBasedOnArgument_unsupported(string command)
                 });
         }
 
+        [Test]
+        [Description("Makes sure getting a git config value does not raise an exception and even if config value is not set it will just return a blank value")]
+        public void ValidateAllConfigOptionsWork()
+        {
+            var settings = _allConfigs.Select(cfg => _referenceRepository.Module.GetEffectiveGitSetting(cfg, false))
+                .OrderBy(s => s.Name)
+                .ToArray();
+            settings.Should().NotContainNulls();
+
+            foreach (var g in settings.GroupBy(s => s.Status))
+            {
+                Console.WriteLine(g.Key?.ToString() ?? "NULL");
+                foreach (var s in g)
+                {
+                    Console.WriteLine($"\t{s}");
+                }
+            }
```

Comment:
Why do we need this?
TBH, I very much dislike it when I run tests the console is getting full of random output. It's not really helpful and often it's extremely difficult to find out where that random output originated from.

Reply:
Console writes can be removed. It was more to show the result. Can wrap if debug.

Reply:
The main point is that I didn't wrap in a try catch and that the test will cause a failure if an exception occurs which should not happen. The select gathering the values and the shoulds are the actual test. I was just using the console outputs to show the items in the test for now.

Reply:
I also use  Devexpress Code Rush Test Runner 
![image](https://github.com/gitextensions/gitextensions/assets/3629489/46de4c32-a748-480d-a40d-734a4909b04a)
So outputs for each test are contained and I can do this 
![image](https://github.com/gitextensions/gitextensions/assets/3629489/21ca7c0c-c13c-479c-8d5c-52fe4ed15fc6)